doc/interop-test-descriptions.md

```diff
@@ -965,13 +969,12 @@ for the `SimpleRequest.response_type`. If the server does not support the
 ### CacheableUnaryCall
 
 Server gets the default SimpleRequest proto as the request. The content of the
-request are ignored. It returns the SimpleResponse proto with the payload set 
-to current timestamp string.  In addition it adds
+request is ignored. It returns the SimpleResponse proto with the payload set
+to current timestamp.  The timestamp is an integer representing current time
```

Comment:
"integer" is still ambiguous, since it is encoded as bytes. If binary, is it big- or little-endian? If not binary, is it ASCII hex, or decimal? If the format doesn't matter, just say the server needs to be consistent.